src/tools/dev/scripts/bv_support/bv_llvm.sh

```diff
@@ -109,82 +127,80 @@ function bv_llvm_dry_run
 
 function apply_llvm_patch
 {
-    info "Currently no patches for llvm"
+    cd ${LLVM_SRC_DIR} || error "Can't cd to LLVM source dir."
+
+#    info "Currently no patches for llvm"
+
+    cd "$START_DIR"
+
+    return 0
 }
 
 function build_llvm
 {
     #
-    # prepare build dir
+    # Uncompress the Clang source dir
     #
-    prepare_build_dir $BV_LLVM_BUILD_DIR $BV_LLVM_FILE
-    untarred_llvm=$?
-    if [[ $untarred_llvm == -1 ]] ; then
-        warn "Unable to prepare LLVM build directory. Giving Up!"
+    uncompress_src_file $CLANG_SRC_DIR $CLANG_FILE
+    untarred_clang=$?
+    if [[ $untarred_clang == -1 ]] ; then
+        warn "Unable to uncompress Clang source file. Giving Up!"
         return 1
+    else
+	# The LLVM build system expects the directory to be named
```

Comment:
Are these LLVM changes in any way related to your *bv cleanup* work? If not, maybe best to do in sep. PR.

Reply:
A bit of rabbit hole pyside was touched which needs llvm so I touched it :-). Especially because someone decided to name all the vars starting with BV_, i.e.  BV_LLVM_VERSION when all others would use LLVM_VERSION.